Doc/faq/programming.rst

```diff
@@ -1164,6 +1164,27 @@ This converts the list into a set, thereby removing duplicates, and then back
 into a list.
 
 
+How do you remove multiple items from a list
+--------------------------------------------
+
+As with removing duplicates, iterating in reverse is one solution::
+
+    for i in range(len(mylist)-1, -1, -1):
+        if remove(mylist[i]):
```

Comment:
I think you might want a different predicate name here, rather than "remove". For beginners, they might think that it actually removes the item. Maybe "should_remove"?

Reply:
Agreed.  Thanks for improvements.